demisto_sdk/commands/init/initiator.py

```diff
@@ -280,9 +329,39 @@ def get_selected_template(self, template: str = "") -> str:
                 )
             return template if template else self.DEFAULT_SCRIPT_TEMPLATE
 
+        elif self.is_pack and self.xsiam:
+            if self.is_modeling_rules:
+                return self.HELLO_WORLD_MODELING_RULES
+            elif self.is_parsing_rules:
+                return self.HELLO_WORLD_PARSING_RULES
+            else:
+                return self.HELLO_WORLD_EVENT_COLLECTOR_INTEGRATION
+
         # if reached here it is a pack init - will be used again if user decides to create an integration
         return template
 
+    def create_files_from_local_template_or_empty(
+        self, file_list: Set[str], path_of_template
+    ) -> None:
+        """Creates empty files according to a given list or from a template if exist.
+
+        Args:
+            file_list (List(str)): A list with path and names of the files.
+        """
+        for file in file_list:
+            file_path = os.path.join(self.full_output_path, file)
+            dir_path = os.path.dirname(file_path)
+            template_path = os.path.join(path_of_template, file)
+            if os.path.exists(template_path) and not os.path.exists(file_path):
+                self.write_to_file_from_template(template_path, file_path)
+            elif not os.path.exists(file_path):
+                if not os.path.exists(dir_path):
+                    os.makedirs(dir_path, exist_ok=True)
+                _, file_extension = os.path.splitext(file_path)
+                with open(file_path, "w") as f:
+                    if file_extension == ".json":
+                        f.write(json.dumps({}, indent=4))
```

Comment:
```suggestion
import os

def process_files(file_list, path_of_template):
    for file in file_list:
        file_path = os.path.join(self.full_output_path, file)
        template_path = os.path.join(path_of_template, file)

        if not os.path.exists(file_path):
            if os.path.exists(template_path):
                self.write_to_file_from_template(template_path, file_path)
            else:
                _, file_extension = os.path.splitext(file_path)
                with open(file_path, "w") as f:
                    if file_extension == ".json":
                        f.write(json.dumps({}, indent=4))

```
These changes make the code more concise, eliminate redundant checks, and improve its readability.